js/src/forum/components/DiscussionPage.js

```diff
@@ -248,10 +218,7 @@ export default class DiscussionPage extends Page {
     // replace it into the window's history and our own history stack.
     const url = app.route.discussion(discussion, (this.near = startNumber));
 
-    this.prevRoute = url;
-    m.route.set(url, null, { replace: true });
```

Comment:
Interesting... So `m.route.set` wasn't really needed here for switching out the url? Interesting that we did so with both `m.route.set` and the native browser API

Reply:
Yes, seems to work as before, I haven't noticed any side effects so far.